content/influxdb/cloud-iox/query-data/flux-sql.md

```diff
@@ -0,0 +1,522 @@
+[`rename()` function](/flux/v0.x/stdlib/universe/rename/).
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs "small" %}}
+[SQL](#)
+[Flux](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```sql
+SELECT time AS _time
+FROM "get-started"
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```js
+// ...
+    |> rename(columns: {time: "_time"})
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+### Unpivot your data
+
+In the context of Flux, data is considered "pivoted" when each field has its own
+column. Flux generally expects a `_field` column that contains the the field key
+and a `_value` column that contains the field. SQL returns each field as a column.
+Depending on your use case and the type of processing you need to do in Flux,
+you may need to "unpivot" your data.
+
+{{< expand-wrapper >}}
+{{% expand "View examples of pivoted and unpivoted data" %}}
+{{% influxdb/custom-timestamps %}}
+
+##### Pivoted data (SQL data model)
+
+| _time                | room    | temp |  hum |
+| :------------------- | :------ | ---: | ---: |
+| 2022-01-01T08:00:00Z | Kitchen |   21 | 35.9 |
+| 2022-01-01T09:00:00Z | Kitchen |   23 | 36.2 |
+| 2022-01-01T10:00:00Z | Kitchen | 22.7 | 36.1 |
+
+##### Unpivoted data (Flux data model)
+
+| _time                | room    | _field | _value |
+| :------------------- | :------ | :----- | -----: |
+| 2022-01-01T08:00:00Z | Kitchen | hum    |   35.9 |
+| 2022-01-01T09:00:00Z | Kitchen | hum    |   36.2 |
+| 2022-01-01T10:00:00Z | Kitchen | hum    |   36.1 |
+
+| _time                | room    | _field | _value |
+| :------------------- | :------ | :----- | -----: |
+| 2022-01-01T08:00:00Z | Kitchen | temp   |     21 |
+| 2022-01-01T09:00:00Z | Kitchen | temp   |     23 |
+| 2022-01-01T10:00:00Z | Kitchen | temp   |   22.7 |
+
+{{% /influxdb/custom-timestamps %}}
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+{{% note %}}
+#### Unpivoting data may not be necessary
+
+Depending on your use case, unpivoting the SQL results may not be necessary.
+For Flux queries that already pivot fields into columns, using SQL to return
+pivoted results will greatly improve the performance of your query.
+{{% /note %}}
+
+To unpivot SQL results:
+
+1. Import the `experimental` package.
+2. [Ensure you have a `_time` column](#rename-the-time-column-to-_time).
+3. Use [`experimental.unpivot()`](/flux/v0.x/stdlib/experimental/unpivot/) to unpivot your data.
+
+```js
+import "experimental"
+import "experimental/iox"
+
+iox.sql(...)
+    |> group(columns: ["room"])
+    |> experimental.unpivot()
+```
+
+### Example SQL query with further Flux processing
+
+{{% influxdb/custom-timestamps %}}
+```js
+import "experimental"
+import "experimental/iox"
+
+query = "
+SELECT
+  time AS _time,
+  room,
+  temp,
+  hum,
+  co
+FROM home
+WHERE
+  time >= '2022-01-01T08:00:00Z'
+  AND time <= '2022-01-01T20:00:00Z' 
+"
+
+iox.sql(bucket: "get-started", query: query)
+    |> group(columns: ["room"])
```

Comment:
Is it worth explaining again why you group by room here when you unpivot?

Reply:
Added this to the section on unpivoting.